ExtractData/PKHeX.ExtractData/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace PKHeX.ExtractData
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string folder = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
+            if (Path.GetFileName(folder).ToLower() == "debug")
```

Comment:
The program was made to launch throught VS. I know is not the best thing to do but because once completed there is only needs to launch it once i prepared the program to find always the files while in the bin/debug folder without copying all the roms to the bin folder. Once the development has been completed because there is no need to run the program anymore there is no need to change the folder to something more portable.